homeassistant/components/myuplink/icons.json

```
@@ -0,0 +1,45 @@
+{
+  "entity": {
+    "binary_sensor": {
+      "elect_add": {
+        "default": "mdi:electric-switch",
+        "state": {
+          "on": "mdi:electric-switch-closed"
+        }
+      }
+    },
+    "number": {
+      "degree_minutes": {
+        "default": "mdi:thermometer-lines"
+      }
+    },
+    "sensor": {
+      "air_flow": {
```

The translation key set in the sensor file is `airflow`, not `air_flow`. Either one needs to be changed, so they match.
```suggestion
      "airflow": {
```